src/db/car/forest.rs

```diff
@@ -116,9 +116,8 @@ impl<ReaderT: super::RandomAccessFileReader> ForestCar<ReaderT> {
         let mut footer_buffer = [0; ForestCarFooter::SIZE];
         cursor.read_exact(&mut footer_buffer)?;
 
-        let footer = ForestCarFooter::try_from_le_bytes(footer_buffer).ok_or(io::Error::new(
-            io::ErrorKind::InvalidData,
-            "Data not recognized as ForestCAR.zst",
+        let footer = ForestCarFooter::try_from_le_bytes(footer_buffer).ok_or(invalid_data(
+            format!("not recognizable as a `{}` file", FOREST_CAR_FILE_EXTENSION),
```

Comment:
```suggestion
            format!("not recognizable as a `{FOREST_CAR_FILE_EXTENSION}` file"),
```

Reply:
I prefer keeping out-of-line for now - it has better integration with `rust-analyzer`